src/oidcop/client_authn.py

```diff
@@ -203,9 +199,7 @@ def verify(self, request, key_type, **kwargs):
         if _sign_alg and _sign_alg.startswith("HS"):
             if key_type == "private_key":
                 raise AttributeError("Wrong key type")
-            keys = _context.keyjar.get(
-                "sig", "oct", ca_jwt["iss"], ca_jwt.jws_header.get("kid")
-            )
+            keys = _context.keyjar.get("sig", "oct", ca_jwt["iss"], ca_jwt.jws_header.get("kid"))
```

Comment:
I'm wondering why you brought all the pep8 modifications back to their original state but if you like it more so for me it's ok and totally relative.

Right it occurs to me that this revision will be more challenging;)

Reply:
Yeah, I shouldn't have run black on it.

Reply:
Not without another commit in between 😜

Reply:
Yeah!